generated/thrift/rpc.thrift

```diff
@@ -144,6 +161,8 @@ struct Block {
 	2: optional Segments segments
 	3: optional Error err
 	4: optional i64 checksum
+	5: optional i64 lastRead
```

Comment:
This has been included so that when a peer bootstraps and blocks are passed between peers we can also pass along last read time.

Reply:
Does this conflict have to be resolved in m3dbnode? Why not resolve it during the report processing.

Reply:
It's not a conflict, peers that do not have a block at all (i.e. bootstrapping nodes) need this data or otherwise it is lost when the block is exchanged.